lib/aws.js

```diff
@@ -35,3 +35,53 @@ module.exports.getS3RVERConfiguration = function() {
         endpoint: new AWS.Endpoint('http://localhost:5000'),
     });
 };
+
+module.exports.uploadToS3 = async function (filePath, isDirectory, bucket, S3FilePath, localPath, callback) {
```

Comment:
Following up on @nicknytko’s comment, `async` functions generally should not take `callback` parameters. I don’t mind whether this is `async`-style or `callback`-style, but it really shouldn’t be both at once.